organizations/user.go

```diff
@@ -232,12 +232,7 @@ func UpdateProfilePicture(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if result.ModifiedCount == 0 {
-		utils.GetError(errors.New("operation failed"), http.StatusInternalServerError, w)
-		return
-	}
-
-	utils.GetSuccess("image updated successfully", nil, w)
+	utils.GetSuccess("Profile picture updated", result, w)
```

Comment:
Why did you do this? the result object has unnecessary details which we have decided to hide and checed that our modification was successful by doing result.ModifiedCount == 0. Please revert. we are returning nil and not the resukt object.